### PR TITLE
KEYCLOAK-16925 Upgrading Guide revised to meet Red Hat standards

### PR DESCRIPTION
**Pull request by andymunro**
@pdrozd One more guide revised to meet Red Hat standards for findability and reusability,

### CODE REVIEW

File: upgrading/topics/upgrade_adapters.adoc [hunk 1]
```diff
@@ -53,17 +55,18 @@ endif::[]
...
 . Download the new adapter archive.
 . Overwrite the keycloak.js file in your application with the keycloak.js file from the downloaded archive.
 
-
 [[_upgrade_nodejs_adapter]]
 == Upgrading the Node.js Adapter
 
 To upgrade a Node.js adapter that has been copied to your web application, complete the following steps:
 
+.Procedure
```
Comment (nehachopra27) [inline upgrading/topics/upgrade_adapters.adoc:70]: @andymunro : Check if there should be two leading lines:
One with "following steps:" and one with "Procedure"

File: upgrading/topics/upgrade_adapters.adoc [hunk 2]
```diff
@@ -53,17 +55,18 @@ endif::[]
...
 To upgrade a JavaScript adapter that has been copied to your web application, complete the following steps: 
 
 . Download the new adapter archive.
 . Overwrite the keycloak.js file in your application with the keycloak.js file from the downloaded archive.
 
-
 [[_upgrade_nodejs_adapter]]
 == Upgrading the Node.js Adapter
```
Comment (nehachopra27) [inline upgrading/topics/upgrade_adapters.adoc]: The adapter casing is inconsistent throughout the file.

File: upgrading/topics/rhsso/changes-73.adoc [hunk 3]
```diff
@@ -116,7 +116,7 @@ set a fixed hostname which makes it easier to make sure the valid hostname is us
 applications to invoke {project_name} through an alternative URL, for example an internal IP address. It is
 recommended that you switch to this approach in production.
 
-=== JavaScipt Adapter Promise
+=== JavaScipt Adapter promise
```
Comment (nehachopra27) [inline upgrading/topics/rhsso/changes-73.adoc]: @andymunro : Please check the casing of Adapter here.

File: upgrading/topics/upgrade_adapters.adoc [hunk 4]
```diff
@@ -53,17 +55,18 @@ endif::[]
 [[_upgrade_js_adapter]]
 == Upgrading the JavaScript Adapter
```
Comment (nehachopra27) [inline upgrading/topics/upgrade_adapters.adoc]: Casing of the adapter is inconsistent in this guide.

File: upgrading/topics/upgrade_adapters.adoc (hunk 2 above)
Comment (nehachopra27) [inline upgrading/topics/upgrade_adapters.adoc]: Casing of adapter.

**Review by nehachopra27 — COMMENTED** [review]
Casing of image titles is high case. Please fix it throughout.

File: upgrading/topics/upgrade_adapters.adoc (hunk 2 above)
Comment (andymunro) [reply upgrading/topics/upgrade_adapters.adoc]: Thanks, I corrected it.

File: upgrading/topics/keycloak/changes.adoc [hunk 5]
```diff
@@ -750,7 +750,7 @@ All Admin REST API endpoints that support pagination now have a default max resu
 ==== `realm-public-key` adapter property not recommended
 
 In 2.3.0 release we added support for Public Key Rotation. When admin rotates the realm keys in Keycloak admin console, the Client
-Adapter will be able to recognize it and automatically download new public key from Keycloak. However this automatic download of new
+adapter will be able to recognize it and automatically download new public key from Keycloak. However this automatic download of new
```
Comment (nehachopra27) [inline upgrading/topics/keycloak/changes.adoc]: Adapter casing has been changed incorrectly.

File: upgrading/topics/keycloak/changes.adoc [hunk 6]
```diff
@@ -811,7 +811,7 @@ The script to add admin users to Keycloak has been renamed to `add-user-keycloak
 
 === Migrating to 1.9.2
 
-==== Adapter option auth-server-url-for-backend-requests removed
+==== adapter option auth-server-url-for-backend-requests removed
```
Comment (nehachopra27) [inline upgrading/topics/keycloak/changes.adoc]: Adapter casing has been changed incorrectly.

File: upgrading/topics/upgrade_adapters.adoc (hunk 1 above)
Comment (andymunro) [reply upgrading/topics/upgrade_adapters.adoc:70]: Thanks, this is correct.  The procedure is introduce first and it is followed by the heading Procedure. An exception is when there are two procedures in one section. Then the heading Procedure comes first.

**Comment by andymunro** [comment]
@nehachopra27 Let me know if this is okay or if it needs more changes. Thanks.

**Comment by andymunro** [comment]
@nehachopra27 Any final comments on this guide?

**Comment by nehachopra27** [comment]
@pdrozd : Changes look good to me

**Review by pdrozd — DISMISSED** [review]

**Comment by andymunro** [comment]
Thanks, @nehachopra27, @pdrozd.

**Comment by andymunro** [comment]
@hmlnarik Would you be able to review this PR, which has been approved by QE?

**Review by abstractj — APPROVED** [review]
LGTM

**Comment by andymunro** [comment]
Thanks, @abstractj   Pavel has approved it also.  Can you merge this?

**Comment by andymunro** [comment]
Thanks, @abstractj

**Pull request merged**